src/main/resources/META-INF/plugin.xml

```diff
@@ -169,6 +169,9 @@
 
         <!-- Structure view -->
         <lang.psiStructureViewFactory language="Robotframework" implementationClass="amailp.intellij.robot.extensions.RobotPsiStructureViewFactory"/>
+
+        <!-- Robot keyword reference -->
+        <psi.referenceContributor implementation="amailp.intellij.robot.psi.reference.KeywordReferenceContributor"/>
```

Comment:
Ok. I am merging your PR locally on my machine, adding a bunch of tests and refactoring a bit. I will then (in a few days) publish a merged branch for you to review, ok?